src/adler/science/Colour.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+from adler.science.PhaseCurve import PhaseCurve
+from adler.utilities.science_utilities import get_df_obs_filt
+
+# TODO: add colour estimate function
+
+
+def col_obs_ref(
+    planetoid,
+    adler_cols,
+    filt_obs="g",
+    filt_ref="r",
+    N_ref=3,
+    x_col="midPointMjdTai",
+    y_col="AbsMag",
+    yerr_col="magErr",
+    x1=None,
+    x2=None,
+):
+    """
+
```

Comment:
This docstring doesn't describe what the function actually does: there should be a short explanation before the Parameters header.